src/layout/PublicApplicationLayout.tsx

```diff
@@ -41,7 +41,8 @@ export default function PublicApplicationLayout() {
         application.status === APPLICATION_STATUS.SUBMITTED ||
         application.status === APPLICATION_STATUS.STARTED
       ) {
-        if (lastSegment !== 'submission-completed') navigate('./submission-completed');
+        if (lastSegment === 'external-onboarding-completed') navigate('./external-onboarding-completed');
```

Comment:
When `lastSegment ===`, shouldn't it just return (like for `change-primary-email`)? Won't this navigate in a loop?

Reply:
Correct, that is better, changed it